server/fleet/hosts.go

```diff
@@ -436,11 +442,17 @@ type AggregatedMDMStatus struct {
 	HostsCount                  int `json:"hosts_count" db:"hosts_count"`
 }
 
+// MDMSolution represents a single MDM solution, as returned by the list hosts
+// endpoint when an MDM Solution ID is provided as filter.
+type MDMSolution struct {
+	ID        uint   `json:"id,omitempty" db:"id"`
+	Name      string `json:"name,omitempty" db:"name"`
```

Comment:
just a thought without much context: do we need `omitempty`? asking because seems like both fields are non-nulleable like in `MunkiIssue`, and that struct doesn't have the `omitempty` tag

Reply:
Yeah you're correct, that's unneeded, I'll remove it.